### PR TITLE
Enable sparse zernikes to be used from ts_wep.

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: python/lsst/donut/viz/zernike_pyramid.py [hunk 1]
```diff
@@ -78,7 +78,9 @@ def zernikePyramid(
     fig: matplotlib Figure
         The figure.
     """
-    jmax = zs.shape[0] + jmin - 1
+    jmin = min(noll_indices)
+    jmax = max(noll_indices)
+    jdict = {x: y for x, y in zip(noll_indices - jmin, range(len(noll_indices)))}
```
Comment (jfcrenshaw) [inline python/lsst/donut/viz/zernike_pyramid.py:83]: Do you need to do `noll_indices - jmin`? It looks like below you only access this with keys `j - jmin`? Why not just let the keys be the Noll indices?

File: python/lsst/donut/viz/zernike_pyramid.py [hunk 2]
```diff
@@ -125,7 +129,7 @@ def shiftAxes(axes, amt):
         scat = ax.scatter(
             xs,
             ys,
-            c=zs[j - jmin],
+            c=zs[jdict[j - jmin]],
```
Comment (jfcrenshaw) [inline python/lsst/donut/viz/zernike_pyramid.py:132]: See note above

**Review by jfcrenshaw — APPROVED** [review]
Looks good. If you haven't already, you should test this on USDF where you specify a sparse set of Zernikes and make sure everything runs and the plots look correct

**Comment by jbkalmbach** [comment]
image

Looks good. Made changes suggested above.

**Pull request merged**